src/client/components/VideoComponent/index.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import './index.scss';
+import userImg from './assets/Oval_2.png';
+import share from './assets/share.png';
+import VideoPlayer from '../VideoPlayerComponent/index';
+
+class VideoComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      'changeupColor': false,
+      'changedownColor': false,
+    };
+  }
+  changeupColor() {
+    this.setState({'changeupColor': !this.state.changeupColor});
+  }
+  changedownColor() {
+    this.setState({'changedownColor': !this.state.changedownColor});
+  }
+  render() {
+    return (
+      <div className="video-component">
+        <VideoPlayer />
+        <div className="video-below">
+          <div className="video-info">
+            <div className="video-name">
+              <span>Midnight Starr - Slow Jam</span>
+              <img className="share" src={share}></img>
```

Comment:
add event listener to this button. Not needed to implement, just pass it from properties